Migrate imap_email_content to config entry setup (#26602)
home-assistant/home-assistant.github.io: source/_integrations/imap_email_content.markdown

The `imap_email_content` integration will read emails from an IMAP email server and report them as a state change within Home Assistant. This is useful if you have a device that only reports its state via email.

{% include integrations/config_flow.md %}

## IMAP server setup

> For the configuration of the IMAP email content sensor The server hostname or IP address of the IMAP server together with username and password are required.

Reviewer: ```suggestion
To configure the IMAP email content sensor, the hostname or IP address of the IMAP server is required, along with a username and password.
```
